src/sentry/tasks/sentry_apps.py

```diff
@@ -37,27 +46,25 @@ def notify_sentry_app(event, futures):
         except SentryAppInstallation.DoesNotExist:
             continue
 
-        payload = app_platform_event('alert', install, data)
-        send_alert_event.delay(sentry_app=sentry_app, payload=payload)
+        data['triggered_rule'] = f.rule.label
+
+        request_data = AppPlatformEvent(
+            resource='event_alert',
+            action='triggered',
+            install=install,
+            data=data,
+        )
+        send_alert_event.delay(sentry_app=sentry_app, request_data=request_data)
```

Comment:
We should be passing basic data structures (`int`, `list`, `dict`, etc.) to `delay`. These values get pickled to be stored on disk (or in memory) while it waits for a worker to pick up the job.

What this is going to do is pickle the entire `sentry_app` instance and `request_data` object. It's going to be a ton of data we store on disk/memory for each of these jobs. 

So for one, we should pass `sentry_app.id` instead of the whole instance itself.

As for `request_data`, we should figure out how to pass the minimum we need to, to instantiate it in the task itself. Maybe the easiest way is to have `AppPlatformEvent` implement some `to_json` and `from_json` function? Alternatively we can pass `resource`, `action`, `installation_id`, and `data` separately, then instantiate it in the task.